pkg/pmem-csi-driver/nodeserver.go

```diff
@@ -587,11 +583,8 @@ func (ns *nodeServer) NodeStageVolume(ctx context.Context, req *csi.NodeStageVol
 		return nil, status.Error(codes.Internal, err.Error())
 	}
 
-	if existingFsType == "" && requestedFsType == "xfs" {
```

Comment:
I think `existingFsType == ""` was too strict here. Suppose mkfs.xfs succeeded but then the function gets interrupted before `xfs_io`. During the next attempt it would find the existing XFS but then skip `xfs_io` although it hasn't been called.

`xfs_io` and now `xfs.ConfigureFS` are idempotent. It's therefore safe to do this step multiple times.

Reply:
yes, this makes sense.